sdk/python/pulumi_kubernetes/_tables.py

```diff
@@ -0,0 +1,913 @@
+    "runAsUser": "run_as_user",
+    "runAsUserName": "run_as_user_name",
+    "runtimeClass": "runtime_class",
+    "runtimeClassName": "runtime_class_name",
+    "runtimeHandler": "runtime_handler",
+    "scaleDown": "scale_down",
+    "scaleIO": "scale_io",
+    "scaleTargetRef": "scale_target_ref",
+    "scaleUp": "scale_up",
+    "schedulerName": "scheduler_name",
+    "scopeName": "scope_name",
+    "scopeSelector": "scope_selector",
+    "seLinux": "se_linux",
+    "seLinuxOptions": "se_linux_options",
+    "secretFile": "secret_file",
+    "secretKeyRef": "secret_key_ref",
+    "secretName": "secret_name",
+    "secretNamespace": "secret_namespace",
+    "secretRef": "secret_ref",
+    "securityContext": "security_context",
+    "selectPolicy": "select_policy",
+    "selfLink": "self_link",
+    "serverAddress": "server_address",
+    "serverAddressByClientCIDRs": "server_address_by_client_cid_rs",
+    "serviceAccount": "service_account",
+    "serviceAccountName": "service_account_name",
+    "serviceAccountToken": "service_account_token",
+    "serviceName": "service_name",
+    "servicePort": "service_port",
+    "sessionAffinity": "session_affinity",
+    "sessionAffinityConfig": "session_affinity_config",
+    "shareName": "share_name",
+    "shareProcessNamespace": "share_process_namespace",
+    "shortNames": "short_names",
+    "sideEffects": "side_effects",
+    "signerName": "signer_name",
+    "singularName": "singular_name",
+    "sizeBytes": "size_bytes",
+    "sizeLimit": "size_limit",
+    "specReplicasPath": "spec_replicas_path",
+    "sslEnabled": "ssl_enabled",
+    "stabilizationWindowSeconds": "stabilization_window_seconds",
+    "startTime": "start_time",
+    "startedAt": "started_at",
+    "startingDeadlineSeconds": "starting_deadline_seconds",
+    "startupProbe": "startup_probe",
+    "statusReplicasPath": "status_replicas_path",
+    "stdinOnce": "stdin_once",
+    "storageClassName": "storage_class_name",
+    "storageMode": "storage_mode",
+    "storagePolicyID": "storage_policy_id",
+    "storagePolicyName": "storage_policy_name",
+    "storagePool": "storage_pool",
+    "storageVersionHash": "storage_version_hash",
+    "storedVersions": "stored_versions",
+    "stringData": "string_data",
+    "subPath": "sub_path",
+    "subPathExpr": "sub_path_expr",
+    "successThreshold": "success_threshold",
+    "successfulJobsHistoryLimit": "successful_jobs_history_limit",
+    "supplementalGroups": "supplemental_groups",
+    "systemUUID": "system_uuid",
+    "targetAverageUtilization": "target_average_utilization",
+    "targetAverageValue": "target_average_value",
+    "targetContainerName": "target_container_name",
+    "targetCPUUtilizationPercentage": "target_cpu_utilization_percentage",
+    "targetPort": "target_port",
+    "targetPortal": "target_portal",
+    "targetRef": "target_ref",
+    "targetSelector": "target_selector",
+    "targetValue": "target_value",
+    "targetWWNs": "target_ww_ns",
```

Comment:
Does this look right?

Reply:
It's the same value we've always emitted:

https://github.com/pulumi/pulumi-kubernetes/blob/c4b1ff2ed580565cd919cdfe7d57d45df4a4e142/sdk/python/pulumi_kubernetes/tables.py#L872

It's not "right," but we keep it the way it's always been (via `PyNameLegacy`) for compatibility.

The strongly typed input/output classes that are landing soon use the "correct" name:

https://github.com/pulumi/pulumi-kubernetes/blob/ea90ce49b0ef9b760b94be47207eb95fd7f2890b/sdk/python/pulumi_kubernetes/core/v1/outputs.py#L2437-L2443

Reply:
Cool, thanks for confirming. Should be safe to merge this then since we aren't actually changing behavior.